doc/Developing.md

```diff
@@ -52,10 +52,8 @@ For Windows instructions [click here](#building-on-windows).
 
 ## Prerequisites
 
- - Node.js `>= 10.11.0` **AND** `< 12.x`.
-   - Preferably, **use** version `10.15.3`, it has the [active LTS](https://github.com/nodejs/Release).
-   - Node.js `11.x` is untested.
-   - Node.js `12.x` is [unsupported](https://github.com/eclipse-theia/theia/issues/5117).
+ - Node.js `>= 12.14.1` **AND** `< 13`.
+   - Preferably, **use** version `12.18.3`, it has the [active LTS](https://nodejs.org/en/).
```

Comment:
Did you check that 12.18.3 work? And there is no issues with node gyp and so on. We tested with 12.14.1 and travis using it as well?

Reply:
Yes you're right, I should list the one that we use during CI (minimum supported electron version) `12.14.1`.

Reply:
that's good, but we should also verify that active LTS actually working, with another PR though is fine